src/Templating/MutableTemplateRegistryAwareInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Twig\Templating;
+
+/**
+ * @author Wojciech Błoszyk <wbloszyk@gmail.com>
+ *
+ * @method MutableTemplateRegistryInterface getTemplateRegistry()
+ * @method bool                             hasTemplateRegistry()
+ * @method void                             setTemplateRegistry(MutableTemplateRegistryInterface $templateRegistry)
+ */
+interface MutableTemplateRegistryAwareInterface
+{
+    // NEXT_MAJOR: uncomment this method
```

Comment:
It's a new interface, you should add this methods now. Do not mind the admin bundle when adding a new interface.